Dockerfile

```
@@ -40,7 +40,7 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-RUN go mod download
+RUN go mod download && go mod tidy
```

```suggestion
RUN go mod download
```
The tidy command here will remove everything since we don't have any go files copied into the container (those are the next steps).